tests/php/ContentReviewBaseTest.php

```diff
@@ -4,8 +4,6 @@
 
 use SilverStripe\Dev\FunctionalTest;
 use SilverStripe\CMS\Model\SiteTree;
-// @todo add translatable namespace
-use Translatable;
```

Comment:
Related to this issue: https://github.com/silverstripe/silverstripe-framework/issues/10548